LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2024 Code Climbers
```

Comment:
Update Code Climbers to be "The Dev Craft LLC"

Reply:
done, thanks for checking!